client/me/help/chat-closure-notice/index.jsx

```diff
@@ -0,0 +1,74 @@
+/**
+ * Fixed notice about upcoming support closures
+ */
+
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import i18n, { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import FormSectionHeading from 'components/forms/form-section-heading';
+import olarkStore from 'lib/olark-store';
+import { title, upcoming, closed } from './messages';
+
+const Notice = localize( ( { translate, closedFrom, closedTo, reason } ) =>
+	<div className="chat-closure-notice">
+		<FormSectionHeading>{ title( { translate, closedFrom, closedTo, reason } ) }</FormSectionHeading>
+		<p>
+			{
+				i18n.moment().isBefore( closedFrom )
```

Comment:
Should we consider to make this condition stricter. e.g. Within one week before `closedFrom`. For the current version, if I modify the date to 10 years back, the closure notice will still show. 

My concern is that with this generalized `ChatClosureNotice` implementation, it is totally possible that we may well drop in all the upcoming closure notice at once. However, if we don't limit this condition, we might show the notices too early and confuse our users.